### DISCUSSION TITLE
[Idea]: Control ptz-camera from M³

### DISCUSSION THREAD

**Discussion by alanfermtz** [sircharlo/meeting-media-manager#1712] [Ideas] [answered]
### Is your idea related to a problem?

We are currently using a low level web camera, so we want to know if its possible to have a ptz camera (or something related) to control and have better digital zoom and be able to move the camera with Obs

### Describe your idea

I have seen plugins for obs but never tried. 
If this works, can be implemented to m3 a ptz gui with the web-socket?

### Describe alternatives you've considered

n/a

**Comment by sircharlo** [comment]
Have you seen [this post](https://github.com/sircharlo/meeting-media-manager/discussions/270#discussioncomment-3114816)  by @mtdvlpr ? Perhaps it could help?

**Comment by mtdvlpr** [comment, marked as answer]
Short answer: already implemented through obs integration! 

Slighty longer answer: when you have multiple obs scenes configured, you'll get a scene picker in the footer of the presentation mode. If you run into problems configuring the ptz-controls plugin, #803 might help. Or [see our documentation](https://sircharlo.github.io/meeting-media-manager/en/#present-media) about configuring obs integration in M³.

**Comment by alanfermtz** [comment]
Thanks for you answers

**Comment by mtdvlpr** [comment]
I'm reopening this so others with similar questions can easily find it :)
